da/test/da_test.go

```diff
@@ -62,14 +63,16 @@ func TestDALC(t *testing.T) {
 	httpServer := startMockCelestiaNodeServer(t)
 	defer httpServer.Stop()
 
+	ctx := context.Background()
+
 	for _, dalc := range registry.RegisteredClients() {
 		t.Run(dalc, func(t *testing.T) {
-			doTestDALC(t, registry.GetClient(dalc))
+			doTestDALC(ctx, t, registry.GetClient(dalc))
 		})
 	}
 }
 
-func doTestDALC(t *testing.T, dalc da.DataAvailabilityLayerClient) {
+func doTestDALC(ctx context.Context, t *testing.T, dalc da.DataAvailabilityLayerClient) {
```

Comment:
Just like in `doTestRetrieve` - I think we can create context inside this method instead of passing it.